_posts/2024-04-21-Prosper_Contibution.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+# If your post title is longer or more complicated
+# than can be represented in the filename, uncomment the following line
+# and specify a custom title
+title:  "Bug Fix Contribution: Electric Vehicle Team"
+
+# Uncomment only one of the below categories
+categories: 
+- Bug Fix
+#- Contribution
+
+
+# Enter your name below
+author: Prosper Omiponle
+---
+
+![EVT Logo](../assets/2024-04-21-Prosper_Contribution/foss_EVT.webp)
+
+The pressures of group projects can be a weird dynamic to navigate. Especially as one very early in my career, it has turned out to be a constant personal experience characterized by a need to prove myself. I joined the RIT Electric Vehicle Team in August 2022. Fresh out of my first Embedded programming class which I thoroughly enjoyed; I was convinced I would easily hit the ground running and in no time, be the go-to guy for all things Firmware on the team. The class was done in pretty much 95% ARM Assembly Language and only the final Lab Exercise had a C component for the main program, but I still thought it would be straightforward to pick up the team's stack which consists of pretty much pure C++.
+
+Suffice to say, that hasn't ended up being the case. However, I've developed a couple features, and done a few bug fixes (That I'm proud of!), so here's my two experience working on a specific bug I fixed last year.
+
+![Figure 1](../assets/2024-04-21-Prosper_Contribution/foss_Fig1.webp)
+Figure 1: Summary clip of Bug Fix
```

Comment:
Since this image is smaller, it seems to want to put your caption on the side of it.

Can you add a newline before it so that it appears on two lines?

```suggestion

Figure 1: Summary clip of Bug Fix
```

Reply:
@pro-sper can you make this fix so i can merge your bug fix blog post?